.deployment.config.json

```diff
@@ -41,27 +45,31 @@
       "id": "deploy-headless-to-s3-version",
       "s3": {
         "bucket": "{terraform.infra.infra.bucket_binaries}",
-        "directory": "proda/StaticCDN/headless/v$[PRERELEASE]",
+        "directory": "proda/StaticCDN/headless/v$[VERSION]",
```

Comment:
`VERSION` is now the full version. We don't need a `PRERELEASE` variable anymore.